src/main/java/net/glowstone/block/itemtype/ItemType.java

```diff
@@ -172,7 +173,7 @@ public Context getContext() {
      */
     public void rightClickBlock(GlowPlayer player, GlowBlock target, BlockFace face,
         ItemStack holding, Vector clickedLoc, EquipmentSlot hand) {
-        if (placeAs != null && (placeAs.getContext() == Context.ANY || placeAs.getContext() == Context.BLOCK)) {
+        if (placeAs != null) {
```

Comment:
There should still be a check here for if the context is not AIR.

Reply:
Fixed.